.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "cSpell.language": "en-US",
+  "editor.rulers": [
+    80
+  ],
+  "git.rebaseWhenSync": true,
+  "python.formatting.provider": "black",
+  "python.jediEnabled": true,
+  "python.linting.flake8Enabled": true,
+  "python.linting.mypyArgs": [
+    "--config-file=tox.ini",
+    "--follow-imports=skip"
+  ],
+  "python.linting.mypyEnabled": true,
+  "python.linting.pydocstyleArgs": [
+    "--match='(?!test_).*\\.py'"
+  ],
+  "python.linting.pydocstyleEnabled": true,
+  "python.linting.pylintCategorySeverity.refactor": "Information",
+  "python.linting.pylintEnabled": true,
+  "python.linting.pylintUseMinimalCheckers": false,
+  "python.pythonPath": "/home/remco/anaconda3/envs/expertsystem/bin/python",
```

Comment:
what to do about this?

Reply:
Oops... :man_facepalming: 81c996d1b8073bae25dc9c82d556909b3e0b9b65

Reply:
But how does this work if everyone defines this line himself (Because that's a workspace setting)? You have to ignore that line somehow. I don't quite understand how this will be handled nicely...

Reply:
Yes this is indeed a slightly problematic point... The way I solve it, is create a [(multi-root) workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces) for experstystem, so that you can specify the python path in the workspace settings and leave the folder settings unaffected.

Reply:
Ok, sounds good. Let's give this a try then, and in case we don't have a good experience with this we can change it. Maybe a default settings file would an option to keep in mind https://stackoverflow.com/a/48387809